Library/Formula/raqm.rb

```diff
@@ -0,0 +1,24 @@
+class Raqm < Formula
+  desc "Library for complex text layout"
+  homepage "https://github.com/HOST-Oman/libraqm"
+  url "https://github.com/HOST-Oman/libraqm/releases/download/v0.1.0/raqm-0.1.0.tar.gz"
+  sha256 "fbccf176ae0ff5aeb73ea3d7a0f2173c432d87a495c10ca2ecdf82323eaa5355"
+
+  depends_on "freetype"
+  depends_on "harfbuzz"
+  depends_on "fribidi"
+  depends_on "glib" => :recommended # for the test suite
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+    system "make", "check"
+  end
+
+  # Raqm does not install any programs
+  test do
+    system "true"
```

Comment:
This doesn’t work as I need the location of FreeType headers, that is why I used `pkg-config` in the first place.